tests/ut/py/test_worker/test_startup_readiness.py

```
@@ -758,6 +761,20 @@ def test_l2_init_then_close_does_not_hang(self, monkeypatch):
             # A second close is a clean no-op (does not re-block on the epoch cv).
             w.close()
 
+    def test_l2_submit_returns_completed_handle(self, monkeypatch):
+        from simpler.task_interface import ChipCallable  # noqa: PLC0415
+
+        w = self._make_l2(monkeypatch)
+        callable = ChipCallable.build(signature=[], func_name="x", binary=b"\x00", children=[])
```

Ruff (0.15.21)

[error] Variable `callable` is shadowing a Python builtin

(A001)